src/tutors/forms/import/xml_file.php

```diff
@@ -21,10 +21,7 @@
     exit;
 }
 
-$errno = -1;
-foreach ($_FILES as $file) {
-    $errno = $file['error'];
-}
+$errno = $_FILES['uploadedfile']['error'];
 
 if ($errno == 0) {
```

Comment:
Should this be if ($errno === 0) {

Reply:
Spoke with Richard. Can't find a definitive answer on what type _error_ should be. Pretty certain it is an int but as I can't be sure, leaving this as a loose comparison just now